module/power_domain/src/mod_power_domain.c

```diff
@@ -754,8 +764,12 @@ static void respond(struct pd_ctx *pd, int resp_status)
         return;
     }
 
+#ifdef BUILD_HAS_NOTIFICATION
```

Comment:
this is not required. fwk_thread_get_delayed_response is separate than notifications

Reply:
yes you're right